http/rest-client-reactive/src/main/resources/modern.properties

```diff
@@ -7,3 +7,4 @@ quarkus.rest-client.logging.scope=request-response
 quarkus.log.category."org.jboss.resteasy.reactive.client.logging".level=DEBUG
 quarkus.http.limits.max-body-size=3G
 quarkus.rest-client.read-timeout = 120000
+quarkus.vertx.warning-exception-time=5s
```

Comment:
There are some warnings in the log related to `o.quarkus.ts.http.restclient.reactive.files.JavaUtils#etSum` used by `/hash` endpoint

The warning looks like an stack trace error because the worker thread is blocked more than `2s` (default value). the warning is fine, is not an error but in our case we are hashing huge files, and is annoying to see the stack trace on the logs when actually is not an error.

Reply:
Yeah, I was planning to fix it together with the failures, but when it turned out to be Quarkus issue, I totally forgot, thanks!

Reply:
I can see in logs the issue is still there :-/, I'll have a look on Monday.